src/dev/pages/button-area/button-area.ejs

```diff
@@ -0,0 +1,46 @@
+<forge-card outlined>
+  <forge-button-area id="button-area">
+    <button slot="button" aria-labelledby="button-area-heading"></button>
+    <div class="flex-wrapper">
+      <div>
+        <div class="forge-typography--headline5" id="button-area-heading">Heading</div>
+        <div>Content</div>
+      </div>
+      <forge-icon-button data-forge-ignore>
+        <button>
+          <forge-icon name="favorite"></forge-icon>
+        </button>
+        <forge-tooltip>Like</forge-tooltip>
+      </forge-icon-button>
+      <forge-icon name="chevron_right"></forge-icon>
+    </div>
+  </forge-button-area>
+</forge-card>
+
+<br />
+
+<forge-card outlined>
+  <forge-expansion-panel id="expansion-panel">
+    <forge-button-area slot="header" id="expansion-panel-button-area">
+      <button slot="button" id="expansion-panel-button" aria-labelledby="expansion-panel-heading" aria-controls="expansion-panel-content" aria-expanded="false"></button>
+      <div class="flex-wrapper">
+        <div>
+          <div class="forge-typography--headline5" id="expansion-panel-heading">Expansion panel</div>
+          <div>Subheading</div>
+        </div>
+        <forge-icon-button data-forge-ignore>
+          <button>
+            <forge-icon name="favorite"></forge-icon>
+          </button>
```

Comment:
I noticed in this demo when tabbing to this inner button and pressing the space key that it toggled the expansion panel, but clicking the button did not. Is that intentional?

Reply:
Yeah, it's demonstrating the ignore attribute